### ISSUE TITLE
Categorize research links in organized markdown file

### ISSUE THREAD

**Issue by wildmolasses** [Great-Hill-Corporation/trueblocks-tokenomics#21]
Include summary from research perspective.

Will look for @tjayrush to post list here.

**Comment by tjayrush** [comment]
This list is 100% random. One of the things we should do is try to find a good way to manage this list including throwing out a bunch of links. These are simply things that I've thought might be interesting at one time or another in no particular order or sense. Let's try to think of a way to better manage this list. I'm not sure a github issue is the best way, but I don't know of any better way. Maybe we can put it in the GitHub wiki. 

Here's the list

https://www.stateofthedapps.com/dapps/quickblocks
https://medium.com/iconominet/iconomi-financial-report-q3-2017-45dc25f30730
BlockSeer is very interesting for Bitcoin.
https://medium.com/dapphub/dapp-tools-and-the-nix-package-manager-c4c692c87310
https://dapphub.com/
https://openbounty.status.im/
https://medium.com/blockchannel/the-ethereum-developer-essential-reading-list-4fe5d788b294
https://coursetro.com/courses/20/Developing-Ethereum-Smart-Contracts-for-Beginners
ethereum/EIPs#721 (comment)
https://www.reddit.com/r/ethereum/comments/7e47y4/ethusd_oracle/
https://medium.com/@simondlr/tokens-2-0-curved-token-bonding-in-curation-markets-1764a2e0bee5
https://blog.propsproject.com/token-economics-for-digital-media-apps-12e16d4c4eb8
https://www.reddit.com/r/ethdev/comments/7f03fz/does_anyone_else_need_a_blockchain_event/ (I can do this trivially)
https://medium.com/the-ethereum-name-service/security-lessons-from-the-ens-launch-c5bc96bf439e
https://auditchain.com/
Code of conduct: https://github.com/paritytech/parity/pull/7157/files
http://cryptobible.io/first-blockchain-smartphone-will-come-preloaded-ethereum-client-status/
http://www.telco.in/#abouttelcoin?telco
https://www.reddit.com/r/ethereum/
https://www.keystats.tech/
https://medium.com/tokenclub/tokenclub-announces-november-token-basket-dd930e671fb
https://www.tokendata.io/
https://github.com/cubedro/eth-net-intelligence-api
https://zipperglobal.com/whitepaper.pdf
https://rufflet.cryptokitties.co/
http://dcarpe.org/
https://www.coindesk.com/forget-token-sales-the-father-of-the-ico-is-all-about-identity-now/
https://venturebeat.com/2017/12/04/blockstack-raises-52-million-to-build-a-parallel-internet-where-you-own-all-your-data/?utm_campaign=Revue%20newsletter&utm_medium=Newsletter&utm_source=Outlier%20Ventures%20Daily%20Brief
https://github.com/etherisc/InsurableCryptoWallet
https://blog.aragon.one/introducing-aragon-nest-1aa8c91c0566
https://medium.com/uport/zug-id-exploring-the-first-publicly-verified-blockchain-identity-38bd0ee3702
https://blog.iota.org/iota-data-marketplace-cb6be463ac7f
https://district0x.io/
See comments in resource intensiveness of listing account transactions https://github.com/gobitfly/etherchain-light
https://medium.com/iex-ec/eli5-iexec-the-cloud-computing-solution-for-blockchain-developers-c988c123bbfb
https://blog.0xproject.com/the-0x-ecosystem-6f2e780567e8
https://medium.com/@maurelian/devcon3-security-round-up-e7754cd30c23
https://towardsdatascience.com/how-docker-can-help-you-become-a-more-effective-data-scientist-7fc048ef91d5
https://blog.cosmos.network/consensus-compare-casper-vs-tendermint-6df154ad56ae
https://iohk.io
paritytech/parity#7262 (comment)
https://medium.com/aigang-network/how-to-read-ethereum-contract-storage-44252c8af925
https://hackmd.io/s/Hk6UiFU7z (Casper)
https://rufflet.cryptokitties.co/
https://blog.chef.io/2009/08/11/why-we-chose-the-apache-license/
https://devpost.com/software/quickblocks
https://github.com/ethereum/wiki/wiki/Sharding-FAQ
https://discuss.ipfs.io/t/ethereum-solidity-smart-contract-ci-toolchain-on-ipfs-wip/1780
http://www.flippening.watch/
https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/credentials.html
https://github.com/zmitton/eth-proof/blob/master/buildProof.js#L147
ethereum/EIPs#777 (comment)
https://blog.havven.io/basecoin-an-algorithmic-central-bank-2fffd164f8c4
https://blog.keep.network/miners-arent-your-friends-cde9b6e0e9ac
https://medium.com/nexus-mutual/smart-contract-cover-55782407722c
https://github.com/decrypto-org/blockchain-papers
https://ethereum.stackexchange.com/questions/3249/how-to-prove-anything-as-mine-on-ethereum/3250#3250
https://medium.com/metamask/scaling-web3-with-signtypeddata-91d6efc8b290
http://blockchainers.org/index.php/2018/02/08/token-erc-comparison-for-fungible-tokens/
http://blockchainers.org/index.php/tag/erc-777/
https://ipld.io/
https://medium.com/@txhash/bridging-the-gap-between-web-2-0-and-blockchain-events-fc77d2c1dcdb
https://medium.com/@codetractio

**Comment by emazurek** [comment]
Added a research folder with a README to the repo. The wiki would be a good place for this too. I don't know.

**Comment by tjayrush** [comment]
Let's put it into a README.md file in the research folder. Easy to edit. We can add a lot of other stuff to that folder. If we want to move it to a wiki later, it's easy enough.

**Comment by emazurek** [comment]
Sounds good. It's in /research/README.md. I'll mark this closed. We can talk about git workflow in #22
